argon2/src/instance.rs

```diff
@@ -8,6 +8,13 @@ use blake2::{
     Blake2b, Digest, VarBlake2b,
 };
 
+#[cfg(feature = "parallel")]
+use {
+    alloc::vec::Vec,
+    core::mem,
```

Comment:
Small nit, but since you have `std` in scope you could do:

```suggestion
    std::{mem, vec::Vec},
```

Reply:
`std` shouldn't be in scope here, I didn't add the `extern crate std` with the last commit 
The need for the direct usage of `std` was a leftover from before we started using rayon

Reply:
Aah ok, never mind then!